api_client/python/timesketch_api_client/search.py

```diff
@@ -1074,8 +1082,10 @@ def scrolling_enable(self):
 
     def to_dict(self):
         """Returns a dict with the respone of the query."""
-        if not self._raw_response:
+        if self._raw_response is None:
             self._execute_query()
+            if not self._raw_response:
+                raise ValueError("No results to return.")
```

Comment:
If _raw_response contains an empty list (no results) will this still evaluate to True? Why not re-use the `is None` clause instead? Same comment for L 1113